README.md

```diff
@@ -1,55 +1,86 @@
-# IBM Blockchain Extension for VSCode
+# IBM Blockchain Platform Extension for VSCode
 <!---Installing instructions
 --->
 <!---Short description of what the extension allows the user to do and key features in bullet points below 
 --->
-The IBM Blockchain extension allows a user to write and test chaincode projects in Visual Studio Code:
-* Generate skeleton chaincode projects
-* Adds a Fabric view to connect and interact with Fabric Blockchain networks
+The IBM Blockchain Platform extension allows a user to easily write and test smart contract projects in Visual Studio Code:
```

Comment:
Preferably change this sentence to:

"The IBM Blockchain Platform extension has been created to assist users in developing, testing, and deploying smart contracts; including connecting to live Hyperledger Fabric environments."